SingularityBase/src/main/java/com/hubspot/singularity/executor/SingularityExecutorLogrotateFrequency.java

```diff
@@ -0,0 +1,19 @@
+package com.hubspot.singularity.executor;
+
+public enum SingularityExecutorLogrotateFrequency {
+    HOURLY("monthly"),
```

Comment:
wouldn't it be far simpler if we made this functionality only work with versons of logrotate which support hourly?